app/controllers/conferences_controller.rb

```diff
@@ -11,6 +11,11 @@ def index
 
   def show; end
 
+  def current
+    current = Conference.first
+    redirect_to "/conferences/#{current.short_title}"
```

Comment:
Don't! Please use the rails url helper http://guides.rubyonrails.org/routing.html